### DISCUSSION TITLE
How best to handle optional slug for homepage?

### DISCUSSION THREAD

**Discussion by howdu** [camya/filament-title-with-slug#8] [Q&A] [answered]
We use this in our CMS where we also have a home/root page where the slug is optional or tried with '/'. 

Overwriting the regex is ok but there doesn't appear to be a way of disabling the required vaildation for specific records. `slugRules` only accepts array and not a closure.

When the slug is '/'  this rtrim removes it:
https://github.com/camya/filament-title-with-slug/blob/7aaf3d075458af2dda4132431609fb4243cf8829/src/Forms/Fields/SlugInput.php#L182

**Comment by camya** [comment]
I added this feature.

1. You have to reset the `required` rule for the slug with `TitleWithSlugInput::make(slugRules: [])` 
2. Than you can insert the `/` char into the the slug field.

You can try it on main. Have fun.

**Comment by camya** [comment, marked as answer]
Added in https://github.com/camya/filament-title-with-slug/releases/tag/v0.5.3
